Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM alpine
+FROM alpine:3.8
 
-# we need ca-certificates for any external https communication
-RUN apk --update upgrade && \
-    apk add curl ca-certificates && \
-    update-ca-certificates && \
+# Adding ca-certificates for external communication, and openssh
+# for attaching to remote nodes
+RUN apk add --update curl ca-certificates openssh-client && \
     rm -rf /var/cache/apk/*
 
 ADD ./build/nomad-helper-linux-amd64 /nomad-helper
-ENTRYPOINT ["/nomad-helper"]
\ No newline at end of file
+RUN chmod +x /nomad-helper
```

Comment:
don't think this is needed, Go should produce a binary that has the +x bit already?

Reply:
I hope it does. 😉  In my testing I added the binary from the releases page. I needed to add the +x there to make it work. I usually prefer making things like these explicit so that I don't ship a broken container because I relied on something in my build environment that ended up failing.